openapi/paths/reports@journal.yaml

```
@@ -0,0 +1,86 @@
+servers:
+  - url: 'https://api-sandbox.rebilly.com/experimental/organizations/{organizationId}'
+    description: Sandbox server
+    variables:
+      organizationId:
+        default: unknown
+        description: Your organization ID.
+  - url: 'https://api.rebilly.com/experimental/organizations/{organizationId}'
+    description: Live server
+    variables:
+      organizationId:
+        default: unknown
+        description: Your organization ID.
+get:
+  x-products:
+    - Reports
+  tags:
+    - Reports
+  summary: Retrieve a journal report
+  operationId: GetJournalReport
+  x-sdk-operation-name: getJournal
+  description: Retrieve a journal report.
+  parameters:
+    - name: currency
+      in: query
+      description: Revenue currency.
+      required: true
+      schema:
+        $ref: ../components/schemas/CurrencyCode.yaml
+    - name: bookedFrom
+      in: query
+      description: Revenue booked from month. Omit to retrieve booked starting from the start.
```

nitpick: I was unsure of the meaning of the second sentence

```suggestion
      description: Month from which revenue is booked. Omit this value to retrieve booked revenue starting from the first booked amount.
```